regparser/grammar/atomic.py

```diff
@@ -9,7 +9,8 @@
 
 lower_p = (
     Suppress("(")
-    + Word(string.ascii_lowercase, max=1).setResultsName("p1")
+    + (Word(string.ascii_lowercase, max=1)
+        | Word("abcdefgh", max=2)).setResultsName("p1")
```

Comment:
Is this to match (aa), (bb), etc?

If so, I've done something similar here: https://github.com/cfpb/regulations-parser/pull/284/files#diff-7e5bfd51f6742e0a7e2a601f0810c257R12

Otherwise these two changes are going to collide when both of these PRs get merged.